### PR TITLE
docs(datepicker): add demo for isOpen state

### PR DESCRIPTION
**Pull request by ghost**
add demo for isOpen state(when browser window is open the datepicker should be open) #3640

### CODE REVIEW

File: demo/src/app/components/+datepicker/demos/is-open/is-open.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class="row">
+  <div class="col-xs-12 col-12 col-md-4 form-group">
+    <input class="form-control" placeholder="Please choose a date" bsDatepicker [isOpen]="isOpen"/>
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/demos/is-open/is-open.html]: Add a button that toggles `isOpen` property

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

File: demo/src/app/components/+datepicker/demos/is-open/is-open.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<div class="row">
+  <div class="col-xs-8 col-8 col-md-5 col-lg-4 form-group">
+    <input class="form-control" placeholder="Please choose a date" #dp="bsDatepicker"  bsDatepicker [isOpen]="isOpen"/>
+  </div>
+  <div class="col-xs-4 col-4 col-md-4 form-group">
+    <input class="btn btn-success" type="button" value="Toggle" (click)="dp.toggle()">
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/demos/is-open/is-open.html]: this button should toggle `isOpen` property

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Needs to fix conflicting files (update your branch from latest development and solve conflicts)

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Needs to fix local conflicts (steps to reproduce - pull pr and update it using same commands as on screenshot)
localmergeconflict

**Comment by ghost** [comment]
update and testing my branch according to Yevheniia comments(conflicts in files) which I can't see

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Pull request merged**